source/reference/mongosync.txt

```diff
@@ -34,15 +34,8 @@ The :ref:`Frequently Asked Questions (FAQ) <c2c-faq>` page addresses questions u
 Compatibility
 -------------
 
-- .. include:: /includes/fact-minimum-server-version-support.rst
-     
-  You can migrate data on clusters (source) with versions of MongoDB
-  lower than 6.0 to an Atlas cluster (destination). Migration from clusters with
-  lower version requires additional preparation and configuration in
-  the clusters with the lower version. `Contact
-  <https://mongodb.com/contact>`__ your account team to inquire about 
-  Professional Services.  
-
+- For details on compatibility and version requirements, see 
+  :ref:`MongoDB Server Version Compatibility<c2c-server-version-compatibility>`.
```

Comment:
Small wording suggestion and formatting nit:

```suggestion
- For details on version compatibility requirements, see 
  :ref:`MongoDB Server Version Compatibility <c2c-server-version-compatibility>`.
```

The space between the text and reference isn't necessary, but helps with readability.